component/otelcol/exporter/prometheus/internal/convert/convert_test.go

```diff
@@ -110,7 +118,27 @@ func TestConverter(t *testing.T) {
 									"count": 333,
 									"sum": 100,
 									"bucket_counts": [0, 111, 0, 222],
-									"explicit_bounds": [0.25, 0.5, 0.75, 1.0]
+									"explicit_bounds": [0.25, 0.5, 0.75, 1.0],
+									"exemplars":[
+										{
+											"time_unix_nano": 1000000001,
+											"as_double": 0.3,
+											"span_id": "aaaaaaaaaaaaaaaa",
+											"trace_id": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+										},
+										{
+											"time_unix_nano": 1000000002,
+											"as_double": 0.5,
+											"span_id": "bbbbbbbbbbbbbbbb",
+											"trace_id": "bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb"
+										},
+										{
+											"time_unix_nano": 1000000003,
+											"as_double": 1.5,
```

Comment:
These are conveniently sorted already too. We should have a test that requires them to be sorted.